core/pkg/monitor/monitor.go

```diff
@@ -38,7 +37,7 @@ func Average(nums []float64) float64 {
 func makeStatsRecord(stats map[string]float64, timeStamp *timestamppb.Timestamp) *service.Record {
 	statsItems := make([]*service.StatsItem, 0, len(stats))
 	for k, v := range stats {
-		jsonData, err := json.Marshal(v)
+		jsonData, err := simplejsonext.Marshal(v)
```

Comment:
i think maybe standard json should be good?

Reply:
yes, you're right

Reply:
done